data/create_roles.sql

```diff
@@ -0,0 +1,6 @@
+CREATE USER aomur_usr;
+CREATE USER fec;
+CREATE USER fec_read;
+CREATE USER openfec_read;
+CREATE USER real_efile;
+CREATE USER real_file;
```

Comment:
Just another note about the `flyway`-specific account to be added here too, if needed.